backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-deleting-backups.adoc

```diff
@@ -0,0 +1,22 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="oadp-deleting-backups-doc"]
+= Deleting backups
+include::_attributes/common-attributes.adoc[]
+:context: deleting-backups
+
+toc::[]
+
+You can delete a backup by creating the `DeleteBackupRequest` custom resource (CR), or by running the `velero backup delete` command as explained in the following procedures. The volume backup artifacts are deleted at different times depending on the backup method. For example, if the backup method is:
```

Comment:
Removing an extra comma, but also, IBM Style Guide says avoid leading a sentence into list items (i.e., use a complete sentence before the colon).

In this case I feel like you could lose the last bit, and just have the previous sentence be the lead-in with the colon, and it works just as well. Might also scan a little easier if it was a new para:
```suggestion
You can delete a backup by creating the `DeleteBackupRequest` custom resource (CR) or by running the `velero backup delete` command as explained in the following procedures.

The volume backup artifacts are deleted at different times depending on the backup method:
```